pkg/harvester-manager/pages/c/_cluster/index.vue

```diff
@@ -1,5 +1,5 @@
 <script>
-import { NAME } from '@shell/config/product/harvester-manager';
+import { NAME } from '~/pkg/harvester-manager/config/harvester-manager';
```

Comment:
`~` will break the currently disabled the `check-plugin` gate

Reply:
done